common/templates/common/sample_djangre.html

```diff
@@ -0,0 +1,13 @@
+{% extends 'base_templates/base.html' %}
+{% load django_react_components %}
+{% load render_bundle from webpack_loader %}
+{% block head %}
+    {% render_bundle 'runtime' %}
+    {% render_bundle 'Home' %}
+    {% render_bundle 'Hello' %}
+{% endblock %}
```

Comment:
:question: I could be wrong, but I don't think these need to be rendered in `<head>`; they can be rendered just before `{% react_component %}` in body. Could you test?

The reason not to put it in `<head>` is that then the page waits for all assets to download before rendering even begins. Moving down later means at least the page DOM starts loading, giving the UX impression that things are loading (instead of hanging).